build/tools/appveyor.bat

```diff
@@ -18,11 +18,12 @@ echo --- Tools versions:
 g++ --version | head -1
 mingw32-make --version | head -1
 ar --version | head -1
+path C:\MinGW\bin;
```

Comment:
No, we need `%path%` for `head`.
To be more precise, we need the `C:\Program Files\Git\usr\bin;` from the path to use `head`. `head` is not part of mingw. But having the git directory in th path also prevents us from removing `SHELL=cmd`. Then it causes the `CreateProcess` error.

I chose to remove everything from path except the mingw directory, but only removing the git directory is also possible [(see my test)](https://ci.appveyor.com/project/MaartenBent/wxwidgets/build/1.0.40):
```cmd
path %path:C:\Program Files\Git\usr\bin;=%
```